#tfArray.msg
nav_msgs/Odometry keyframe
geometry_msgs/TransformStamped[] transforms
std_msgs/Float64MultiArray[] covariances